src/nwb_conversion_tools/tools/roiextractors/roiextractors.py

```diff
@@ -80,7 +134,7 @@ def add_devices(nwbfile: NWBFile, metadata: dict):
     return nwbfile
 
 
-def add_two_photon_series(imaging, nwbfile, metadata, buffer_size=10, use_times=False):
+def add_two_photon_series(imaging, nwbfile, metadata, buffer_size=10):
```

Comment:
Can we add type hints since it is being refactored?

Reply:
Yes, good idea. We need to add unit-tests for this at some point as well and a doc-string. But right now I just wanted to get rid of the `use_times` value as `roieextractors` can can decide that on the spot according to what's available.

Reply:
Maybe keep it but at a deprecation warning that informs on run that the argument is no longer needed? (can use the same August deadline as the other deprecation)

Reply:
Sure, why not?
It is back with a warning in the last commit. 
Feel free to add a more descriptive warning if you feel like : )